Content.Client/CharacterInterface/CharacterInterfaceComponent.cs

```diff
@@ -37,16 +36,16 @@ public CharacterWindow(List<ICharacterUI> windowComponents)
             {
                 Title = "Character";
 
-                var contentsVBox = new BoxContainer
```

Comment:
There should actually be a BoxContainer inside the ScrollContainer. This is because the BoxContainer stacks Controls while the ScrollContainer just overlaps them.